cocoa/tests_backend/window.py

```diff
@@ -27,16 +30,34 @@ async def wait_for_window(
         message,
         minimize=False,
         full_screen=False,
-        state_switch_not_from_normal=False,
+        expected_state=None,
```

Comment:
Are we still using the `minimize` and `full_screen` arguments? If so... why? My original question about this was "What's the difference between `full_screen=True` and `expected_state=FULL_SCREEN`" - if the answer is "nothing", then why do we need the other arguments?

Reply:
Looks like, I had missed cleanup of unused parameters. I have removed them now.